### PR TITLE
[PAY-853] Premium content fixes on web

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Fix some bugs introduced in premium PRs and add more functionality:
- poll for premium track access on unlock
- handle follow/unfollow, unlock/lock of follow-gated  tracks
- update unlocking state as per design
- sort nft collections in track upload
- disable clicking/playing locked premium track
- fix playlist tile hover icon bug
- update copy
- some refactor (more to come)

### Dragons

Touches tracks, but should be okay. also gated behind feature flags.

### How Has This Been Tested?

Local dapp vs stage and vs prod

### How will this change be monitored?

N/A

### Feature Flags ###

Existing premium content feature flags

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc-more

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc-more

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc-more

File: packages/web/src/common/store/premiumContent/sagas.ts [hunk 1]
```diff
@@ -293,47 +294,48 @@ function* updateCollectibleGatedTrackAccess(
   }
 }
 
-const PREMIUM_TRACK_POLL_FREQUENCY = 2000
+const PREMIUM_TRACK_POLL_FREQUENCY = 5000
```
Comment (sddioulde) [inline packages/web/src/common/store/premiumContent/sagas.ts:297]: i see that it takes much longer than 2 seconds, so decreasing the frequency here to reduce number of requests to DNs

File: packages/web/src/components/artist/ArtistCard.tsx [hunk 2]
```diff
@@ -83,7 +84,10 @@ export const ArtistCard = (props: ArtistCardProps) => {
   const handleUnfollow = useCallback(() => {
     dispatch(unfollowUser(user_id, FollowSource.HOVER_TILE))
     dispatch(setNotificationSubscription(user_id, false, true))
-  }, [dispatch, user_id])
+    if (onUnfollow) {
+      onUnfollow()
```
Comment (rickyrombo) [inline packages/web/src/components/artist/ArtistCard.tsx:88]: nit: could do `onUnfollow?.()`

File: packages/web/src/components/track-availability-modal/TrackAvailabilityModal.module.css [hunk 3]
```diff
@@ -142,7 +142,7 @@
 
 .collectibleGated {
   padding: 0;
-  margin-top: 16px;
+  margin-top: 24px;
```
Comment (rickyrombo) [inline packages/web/src/components/track-availability-modal/TrackAvailabilityModal.module.css:145]: nit: `var(--unit-6)`

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc-more

File: packages/web/src/components/artist/ArtistCard.tsx (hunk 2 above)
Comment (sddioulde) [reply packages/web/src/components/artist/ArtistCard.tsx:88]: yes. was debating readability. maybe a client sync item?

File: packages/web/src/components/track-availability-modal/TrackAvailabilityModal.module.css (hunk 3 above)
Comment (sddioulde) [reply packages/web/src/components/track-availability-modal/TrackAvailabilityModal.module.css:145]: thanks!

**Pull request merged**